src/reference/cog_server_configuration.adoc

```diff
@@ -45,6 +45,17 @@ These are intended for automated installations of Cog (e.g., using management so
 *  `COG_BOOTSTRAP_PASSWORD`
 *  `COG_BOOTSTRAP_USERNAME`
 
+[[COG_CUSTOM_TEMPLATE_DIR]]`COG_CUSTOM_TEMPLATE_DIR`::
+The path to your custom template directory.
++
+Cog uses a set of common templates to format responses generated internally. Some of these templates may be overridden by placing a custom version in a directory and setting this var to said directory's path.
++
+[TIP]
```

Comment:
I don't think we need a full-blown TIP call-out here. A simple `See <<Customizing the standard error template>> for more information.` would suffice.

Saying that `error` is the only one that can be overridden would probably best be left to that more detailed page; that way we only need to keep a single list of templates, rather than multiple.